.github/workflows/main.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ghc-version: ['8.10', '8.8']
+        ghc-version: ['8.10', '9.4.7']
```

Comment:
@DavidMazarro Thank you for your efforts on this. I believe we should also increase the `base` version constraint and update the `tested-with` section in all cabal files. Additionally, if we raise the lower bound for `base` we should also release this as a major change.

Reply:
Additionally, consider changing this list to the `-` style to make future comparisons easier:

```yml
ghc-version:
  - '9.4' # Let's only specify the major and minor versions to allow patch updates
  - '8.10'
```

Reply:
Let's keep all three versions (8.8, 8.10, 9.4) for now in the build matrix. If we ever need to drop support for GHC 8.8, we can revisit this topic (and create a major version bumping `base`).